public/css/style.css

```diff
@@ -990,7 +990,7 @@ figure.ethereum-logo {
 	background: #FFF;	
 }
 
-.main-tutorial code:after {
+.main-tutorial pre:after {
```

Comment:
We should remove that, as it moves when you scroll. I didn't find a way (besides adding more wrapper divs) to make it not scroll